telnyx_rtc/src/test/java/com/telnyx/webrtc/sdk/socket/TxSocketTest.kt

```diff
@@ -169,10 +173,10 @@ class TxSocketTest : BaseTest() {
         // Sleep to give time to connect
         Thread.sleep(6000)
         verify(client, times(1)).onConnectionEstablished()
-        client.pingPong()
+       /* client.pingPong()
```

Comment:
@RakshakGaind0910 if you think that test should not contain the pingpong check, please remove it rather than comment out. cc: @enzoqtvf